src/editor/codemirror/language-server/documentation.ts

```diff
@@ -52,8 +52,9 @@ const fixupMarkdown = (input: string): string => {
   // Messy because it's after escaping. Fragile because it's regex.
   // Let's see if we can upstream or align the docs with supported syntax.
   return input
-    .replace(/^\\\n/, "")
     .replace(/`([\w² \n]+?) ?<(.*?)>`\\_/gs, "[$1]($2)")
+    .replace(/\\\*args/, "*args")
+    .replace(/\\\*kwargs/, "*kwargs")
```

Comment:
Need to check this was required.

This whole thing is pretty fragile, but I think less costly than improving the rEST -> Markdown code in Pyright. Worth revisiting if/when we use third-party docstrings.

Reply:
Required to avoid leading slash in the *args parameter.